site/static/js/main.js

```diff
@@ -472,3 +507,5 @@ document.addEventListener('s72loaded', function(event) {
   let app = event.detail.app;
   documentReady(app);
 });
+
+window.onresize = togglePageTopPadding;
```

Comment:
Please use `window.addEventListener` and move it into `documentReady()`

Reply:
You should also do your best to ensure this function isn't called every cpu cycle as the user resizes.
Without any type of throttling this function will run 100s of times per resize.

The slider and - maybe - carousel have examples on throttling resize logic.